db/migrations/0002_alter_skill_race.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2024-11-06 09:08
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='skill',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='skill_set', to='db.Race'),

```

Comment:
The `to` parameter in the ForeignKey should reference the correct app name. Replace `'db.Race'` with the correct app name, e.g., `'your_app_name.Race'`.